src/App.js

```diff
@@ -10,6 +11,8 @@ function App() {
             <Sidebar />
             <Routes>
                 <Route path="/" element={<HomePage />} />
+                <Route path="/login" element={<LoginPage />} />
```

Comment:
Possibly add <requireUnauth>...</>
so sign in and signup can be diverted to the homepage in case the user is logged in.

Reply:
yeah considering it in  future PR. Will make one Restricted Route that checks whether user is logged In and as soon as user login or signup will redirect to homepage.